examples/python-dataflow/dataflow_yolo.yml

```diff
@@ -0,0 +1,44 @@
+nodes:
+  - id: camera
+    build: pip install ../../node-hub/opencv-video-capture
+    path: opencv-video-capture
+    inputs:
+      tick: plot/tick
+
```

Comment:
So I find it confusing to have empty lines in between nodes.

I would prefer that we keep empty lines for new node.